goentri.com.squareup.json

```diff
@@ -3,25 +3,26 @@
     "providerName": "Entri",
     "serviceId": "squareup",
     "serviceName": "Squareup",
-    "version": 1,
+    "version": 2,
     "logoUrl": "https://cdn.goentri.com/logo.svg",
     "description": "Allows user to easily set up domain using Entri",
     "syncPubKeyDomain": "goentri.com",
     "syncRedirectDomain": "api.goentri.com, goentri.com, entri.com, app.entri.com,app.goentri.com",
-    "variableDescription":"ip is the ip to point the domain",
+    "variableDescription":"ip is the ip to point the domain, aHost is whe A record is pointed, cnameHost is the host for the CNAME record and cnamePointsTo is where the CNAME is pointed ",
+    "hostRequired": true,
```

Comment:
I'm really confused by this template.
If you set hostRequired it means you expect always to set the host variable and generate a subdomain.
Then `%aHost%` and `%cnameHost%` will generate a subhost of this subdomain each. Sure about it?
The v1 of the template looked like a typical main domains + www subdomain setup.

Reply:
Hey Pawel, I've made new changes to avoid the behavior you reported above.

Reply:
Hi Pawel,

I've made some changes to avoid this behavior you reported above.

Thank you.